components/todo-input.js

```diff
@@ -1,20 +1,110 @@
-import React from "react";
+import React, { useState } from 'react';
+
 import styled from "styled-components";
 
-export default function TodoInput() {
+export default function TodoInput({addTodo}){
+
+  const [newTodo, setNewTodo] = useState({});
+
+  const changeInputData = (e) => {
+    setNewTodo({...newTodo, [e.target.name]:e.target.value});
+  }
+
   return (
-    <Wrapper>
-      리액트-투두
-      <br />
-      시간을 입력하는 칸
-      <br />
-      Todo를 입력하는 칸
-    </Wrapper>
+    <InputWrapper>
+      <TimeBox>
+        <Time>시간</Time>
+        <InputDay name="date" type="number" placeholder="날짜를 입력하세요 (ex.20200404)" 
+        onInput = {(e) =>{e.target.value = Math.max(0, parseInt(e.target.value) ).toString().slice(0,8)}}
+        onChange={changeInputData}
+        />
+      </TimeBox>
+      <TodoBox>
+        <Todo>TODO</Todo>
+        <InputTodo name="content" placeholder='할 일을 입력하세요' onChange={changeInputData}></InputTodo>
+      </TodoBox>
+      <EnterBtn onClick={() => addTodo({...newTodo})}>등록</EnterBtn>
+    </InputWrapper>
   );
 }
 
-const Wrapper = styled.div`
-  border: solid 1px;
-  font-size: 18px;
-  flex: 1;
+
+const InputWrapper = styled.div`
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  -webkit-box-align: center;
+  align-items: center;
+  height: 37rem;
+  flex: 1 1 0%;
+  border-width: 1px;
+  border-style: solid;
+  border-color: black;
+  border-image: initial;
+  padding: 1rem 2rem;
+`;
+
+const TimeBox = styled.div`
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  -webkit-box-pack: justify;
+  justify-content: space-between;
+  align-items: flex-start;
+  margin-bottom: 3rem;
+`;
+
+const Time = styled.p`
+  font-size: 1.5rem;
+  padding: 0px;
+  margin: 0px;
 `;
+
+const InputDay= styled.input`
+  width: 80%;
+  border-width: 1px;
+  border-style: solid;
+  border-color: rgb(97, 97, 97);
+  border-image: initial;
+  padding: 0.5rem 0.8rem;
+`;
+
+const TodoBox= styled.div`
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  -webkit-box-pack: justify;
+  justify-content: space-between;
+  align-items: flex-start;
+  margin-bottom: 3rem;
+`;
+
+const Todo = styled.p`
+  font-size: 1.5rem;
+  padding: 0px;
+  margin: 0px;
+`;
+
+const InputTodo = styled.textarea`
+  width: 80%;
+  height: 30vh;
+  resize: none;
+  border-width: 1px;
+  border-style: solid;
+  border-color: rgb(97, 97, 97);
+  border-image: initial;
+  padding: 0.5rem 0.8rem;
+`;
+
+const EnterBtn = styled.button`
+  color : white;
+  background-color: rgb(97, 97, 97);
+  font-size: 1.5rem;
+  outline: none;
+  border-width: initial;
+  border-style: none;
+  border-color: initial;
+  border-image: initial;
+  padding: 0.5rem 1rem;
+  border-radius: 0.3rem;
+`;
```

Comment:
```suggestion
const SubmitButton = styled.button`
  color : white;
  background-color: rgb(97, 97, 97);
  font-size: 1.5rem;
  outline: none;
  border: none;
  padding: 0.5rem 1rem;
  border-radius: 0.3rem;
`;
```
1. input, textarea 값을 제출한다는 의미에서 enter보단 submit을 써주시는 게 좋아요.
또한 컴포넌트 네이밍은 너무 길어지지 않을 때를 제외하고는 최대한 풀어서 써주시는 게 좋습니다.
```EnterBtn```을 ```SubmitButton```으로 바꿔주시면 좋을 것 같아요.
2. 버튼의 테두리가 없기 때문에 ```border: none;``` 써주시면 될 것 같아요.